### PR TITLE
Test/stabilization

### PR DESCRIPTION
**Pull request by infojunkie**
Creating this PR to be able to leave comments

### CODE REVIEW

File: test/spec/app_spec.rb [hunk 1]
```diff
@@ -55,6 +53,7 @@
 
   before :each do |example|
     $caller_name = example.metadata[:description_args]
+    p $caller_name
```
Comment (infojunkie) [inline test/spec/app_spec.rb]: Remove debugging output

File: test/spec/app_spec_helpers.rb [hunk 2]
```diff
@@ -101,8 +101,41 @@ def slack_login
...
+    element
+  end
+
+  def wait_for_selector_list(selector, type = :css, timeout = 30)
+    @wait = Selenium::WebDriver::Wait.new(timeout: timeout)
+    elements = @wait.until { @driver.find_elements(type, selector) }
+    elements
+  end
+
+  def wait_txt_change(txt, selector, type = :css)
```
Comment (infojunkie) [inline test/spec/app_spec_helpers.rb]: Rename to `wait_for_text_change`

File: test/spec/app_spec_helpers.rb [hunk 3]
```diff
@@ -101,8 +101,41 @@ def slack_login
...
+    count = 0
+    begin
+      count = count + 1
+      el = wait_for_selector(selector, type)
+      sleep 1
+    end while (el.text == txt and count < 10)
```
Comment (infojunkie) [inline test/spec/app_spec_helpers.rb]: `count` should probably be passed by the client

File: test/spec/app_spec_helpers.rb [hunk 4]
```diff
@@ -101,8 +101,41 @@ def slack_login
...
+    el.text
+  end
+
+  def wait_size_change(s, selector, type = :css)
+    count = 0
+    begin
+      count = count + 1
+      el = wait_for_selector_list(selector, type)
+      sleep 1
+    end while (s == el.size and count < 10)
```
Comment (infojunkie) [inline test/spec/app_spec_helpers.rb]: Rename to `wait_for_size_change` and add `count` parameter with default = 10

File: test/spec/app_spec_helpers.rb (hunk 2 above)
Comment (clarissacastella) [reply test/spec/app_spec_helpers.rb]: Ok.

File: test/spec/app_spec_helpers.rb (hunk 3 above)
Comment (clarissacastella) [reply test/spec/app_spec_helpers.rb]: Ok

File: test/spec/app_spec_helpers.rb (hunk 4 above)
Comment (clarissacastella) [reply test/spec/app_spec_helpers.rb]: Ok

File: test/spec/app_spec.rb (hunk 1 above)
Comment (clarissacastella) [reply test/spec/app_spec.rb]: Until the end of the day, when finishing a stable version

Comment (clarissacastella) [reply test/spec/app_spec.rb]: Removed

File: test/spec/app_spec_helpers.rb [hunk 5]
```diff
@@ -8,6 +8,23 @@ def get_element(selector, type = :css)
     }
   end
 
+  def time_handle(selector, value, type = :css, visible = true)
```
Comment (caiosba) [inline test/spec/app_spec_helpers.rb]: Is this method used anywhere? I didn't find usage on this diff.

Comment (clarissacastella) [reply test/spec/app_spec_helpers.rb]: No, can be removed.

**Pull request merged**